deploy/batch-rca-automation/batch_rca_headless.sh

```
@@ -0,0 +1,336 @@
+# Fetch logs using logs-fetcher script from plugin cache
+# Assumes: REMOTE_HOST, REMOTE_DIR, JOB_LOGS_DIR are set in env
+LOGS_FETCHER_SCRIPT=$(find ~/.claude/plugins/cache -name "fetch_logs_ssh.py" 2>/dev/null | head -1)
+
+if [ -z "$LOGS_FETCHER_SCRIPT" ]; then
+  echo "[ERROR] logs-fetcher plugin not found. Please install aiops-plugin."
+  exit 1
+fi
+
+# First, get the list of files to fetch from remote (for job ID extraction)
+FILE_LIST_CMD="cd ${REMOTE_DIR} && find . -maxdepth 1 -type f -name '*.transform-processed' -newermt '${START_TIME}' -printf '%T@ %f\\n' | sort -rn | cut -d' ' -f2- | head -n ${LIMIT}"
+SSH_RAW_OUTPUT=$(ssh "$REMOTE_HOST" "$FILE_LIST_CMD" 2>&1)
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)